ansible_role/templates/broker.servicecatalog.yaml

```diff
@@ -37,11 +37,11 @@ spec:
   caBundle: {{
     lookup(
       'k8s',
-      kind='Secret',
+      kind='ConfigMap',
```

Comment:
If there is a more specific API we could find from the cert-signer operator, we should use that to determine if the `caBundle` should come from the secret or configmap.

Reply:
updated